polling_stations/settings/constants/uploads.py

```diff
@@ -3,3 +3,8 @@
 GITHUB_USERNAME = "polling-bot-4000"
 GITHUB_EMAIL = "developers@democracyclub.org.uk"
 GITHUB_REPO = "DemocracyClub/UK-Polling-Stations"
+
+CUSTOM_DOMAINS = {
+    "custom_council.gov.uk": "FOO",
+    "custom_council2.gov.uk": "BAR",
```

Comment:
This should be empty with a comment to explain the expected format (email domain (i.e. the bit after '@') to reg code?) and how to use it preferably with a real world example.